### ISSUE TITLE
Misleading getblockchaininfo doc

### ISSUE THREAD

**Issue by bip-42** [bitcoin/bitcoin#17512]
The help text on the `getblockchaininfo` RPC states
```
getblockchaininfo
Returns an object containing various state info regarding blockchain processing.

Result:
{
 "blocks": the current number of blocks processed in the server
 "headers": the current number of headers we have validated
...
}
```

However, for both `blocks` and `headers` it returns the current height, i.e. #blocks+1.

This can be especially confusing on `regtest` when working with small block numbers.

**Comment by MarcoFalke** [comment]
This has been fixed in fab0c820fa4c0c3227eec85c64310a3bf938a149

**Comment by MarcoFalke** [comment]
So it returns "`the height of the most-work fully-validated chain. The genesis block has height 0`".
